src/main/resources/component-kernel.conf

```diff
@@ -128,6 +128,13 @@ cache {
 
 }
 
+xpath.namespaces {
```

Comment:
Used by StructuredNode (via ConfigBasedNamespaceContext) to resolve namespace
prefixes within XPATH queries into proper namespace URIs.